src/frontend/src/pages/PIAIntakeForm/index.tsx

```diff
@@ -474,7 +476,22 @@ const PIAFormPage = () => {
 
     return () => clearTimeout(autoSaveTimer);
   });
-
+  const handleBack = () => {
+    navigate(-1);
+  };
+  const handleNext = () => {
+    const editMode = pathname.split('/')[4];
```

Comment:
splitting the 5th item in a list is dangerous. We may need to sync up to resolve this. I remember doing something on those lines.

Reply:
Personally, I would avoid doing it this way. The other way we pop it from the link. Not sure I like both ideas. 
I would set an edit flag in the router when we are in edit mode and make all the logic based on that flag rather than taking it from the link. My 2 cents.

Reply:
Agree, that is my thought as well. All redirects happen from the router.